NHSD.BuyingCatalogue.Ordering.Api.sln

```diff
@@ -1,96 +1,103 @@
-﻿
```

Comment:
I wonder what caused this to replace the entire file

Reply:
Who knows, what would you like me to do?